lib/src/main/java/org/team100/lib/hid/JoystickControl.java

```diff
@@ -49,7 +49,7 @@ public Twist2d twist() {
                 deadband(-1.0 * clamp(m_controller.getY(), 1), m_config.kDeadband, 1),
                 m_config.kExpo);
         double dy = expo(
-                deadband(-1.0 * clamp(m_controller.getX(), 1), m_config.kDeadband, 1),
+                deadband(1.0 * clamp(m_controller.getX(), 1), m_config.kDeadband, 1),
```

Comment:
so if FRC positive Y is to the left, and Xbox positive Y is to the right, so i think the inversion you removed was actually correct.  in the past we've made this mistake (several times) because we do things like map the corners of the drivetrain incorrectly or something, and inverting the control "fixes" it.  but it's not fixed.